Bottom: 6d31c8df8055ba834af8a671d76385948fae3631
Top:    bd5b90d1e3159570e9c11c98b54213e89d611b36
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:31:43 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 2c19752c..e1abd2b7 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -137,7 +137,7 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   remote_user: ansibleremote
   become: true
   become_user: root
@@ -145,14 +145,7 @@
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
